.drone.env

```diff
@@ -1,3 +1,3 @@
 # The version of OCIS to use in pipelines that test against OCIS
-OCIS_COMMITID=25d6e4efd119049f290683d6d126185368058b4e
+OCIS_COMMITID=5aeb76f6c8aee574fdd2710f9e883efe0c8166f8
```

Comment:
Could you rebase this branch? The commit ID you're referring here is already on web master :)